schema/anyof.go

```diff
@@ -68,3 +73,27 @@ func (v AnyOf) Serialize(value interface{}) (interface{}, error) {
 
 	return value, nil
 }
+
+// LessFunc implements the FieldComparator interface, and returns either the
+// first non-nil LessFunc or nil.
+func (v AnyOf) LessFunc() (LessFunc, bool) {
```

Comment:
PS! There is a noticeable limitation with this design. Consider the case:

```go
"myField": {
  Default: null,
  Validator: &schema.AnyOf{
    &schema.String{},
    &schema.Float{},
  },
}
```

The query parse step would now allow a query `{myField:{&gt:null}}`, even though `null` is not strictly allowed for a compare operation; that is because `AnyOf` actually should use different criteria (consider only `schema.FieldCompirator` implementations), when validating a query value for a conspirator operator.

PErhaps we can allow this limitation for now (and for v0.2). If I have time during Christmas, I am planning to have a look at forking the `schema` package to a new repo and try to come up with a design that can address #77. I want to do this in a separate repo, so I can experiment with designs without worrying about breaking things.

Reply:
Thanks, it seems acceptable.